docs/getting-started/contributors.md

```diff
@@ -209,24 +242,125 @@ getEntity is now a function that returns a Promise.
+    locationService,
++   fluxCapacitor,
+    logger: env.logger,
+    config: env.config,
+  });
+```
+````
+
+## Developer Certificate of Origin
+
+As with other CNCF projects, Backstage has adopted a [Developers Certificate of Origin (DCO)](https://developercertificate.org/). A DCO is a lightweight way for a developer to certify that they wrote or otherwise have the right to submit code or documentation to a project.
+
+To certify the code you submit to the repository you'll need to add a `Signed-off-by` line to your commits.
+
+`$ git commit -s -m 'Awesome commit message'`
+
+Which will look something like the following in the repo;
+
+```text
+Awesome commit message
+
+Signed-off-by: Jane Smith <jane.smith@example.com>
+```
+
+- In case you forgot to add it to the most recent commit, use `git commit --amend --signoff`
+- In case you forgot to add it to the last N commits in your branch, use `git rebase --signoff HEAD~N` and replace N with the number of new commits you created in your branch.
+- If you have a very deep branch with a lot of commits, run `git rebase -i --signoff $(git merge-base -a master HEAD)`, double check to make sense of the commits (keep all lines as `pick`) and save and close the editor. This should bulk sign all the commits in your PR. Do be careful though. If you have a complex flow with a lot of branching and re-merging of work branches and stuff, merge-base may not be the right solution for you.
+
+Note: If you have already pushed your branch to a remote, you might have to force push: `git push -f` after the rebase.
+
+### Using GitHub Desktop?
+
+If you are using the GitHub Desktop client, you need to manually add the `Signed-off-by` line to the Description field on the Changes tab before committing:
+
+```text
+Awesome description (commit message)
+
+Signed-off-by: Jane Smith <jane.smith@example.com>
+```
+
+In case you forgot to add the line to your most recent commit, you can amend the commit message from the History tab before pushing your branch (GitHub Desktop 2.9 or later).
+
+## API Reports
+
+Backstage uses [API Extractor](https://api-extractor.com/) and TSDoc comments to generate API Reports in Markdown format. These reports are what drive the [API Reference documentation](https://backstage.io/docs/reference/). What this means is that if you are making changes to the API or adding a new plugin then you will need either generate a new API Report or update an existing API Report. If you don't do this the CI build will fail when you create your Pull Request.
+
+There are two ways you can do this:
+
+1. You can run `yarn build:api-reports` from the root of the project and it will go through all of the existing API Reports and update them or create new ones as needed. This may take a while but is generally the best method if you are new to this.
+2. You can run `yarn build:api-reports plugins/<your-plugin-with-changes>` from the root and it will update the existing API Report or create a new one
```

Comment:
```suggestion
2. You can run `yarn build:api-reports plugins/<your-plugin-with-changes>` from the root and it will update the existing API Report or create a new one.
```

Reply:
Fixed 👍